### ISSUE TITLE
Errors when running tests

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#1562]
Seeing this error and variants on it when I run tests from the `release-0.18.0` branch

```
I20161110-13:18:22.130(8)? Exception in queued task: TypeError: Cannot set property '__revisions' of undefined
I20161110-13:18:22.130(8)?     at Object.removed (server/publications/collections/products.js:289:13)
```

**Comment by zenweasel** [comment]
@mikemurray probably already knew this but it appears to be a bug in `publication-collector`. I have submitted a [PR](https://github.com/johanbrook/meteor-publication-collector/pull/13) .

**Comment by aaronjudd** [comment]
latest publication-collector is now in 0.18, with @zenweasel's PR.

**Comment by zenweasel** [comment]
Closed via #1603
